common/txmgr/broadcaster.go

```diff
@@ -558,21 +558,35 @@ func (eb *Broadcaster[CHAIN_ID, HEAD, ADDR, TX_HASH, BLOCK_HASH, SEQ, FEE]) hand
 		eb.sequenceTracker.GenerateNextSequence(etx.FromAddress, *etx.Sequence)
 		return err, true
 	case client.Underpriced:
-		return eb.tryAgainBumpingGas(ctx, lgr, err, etx, attempt, initialBroadcastAt, retryCount+1)
+		bumpedAttempt, retryable, newErr := eb.replaceAttemptWithBumpedGas(ctx, lgr, err, etx, attempt)
+		if newErr != nil {
+			err = fmt.Errorf("%v, failed to create bumped attempt, %v", err, newErr)
+			return err, retryable
+		}
```

Comment:
nit: I think this error message might not be accurate since there's other stuff that could have failed inside of `replaceAttemptWithBumpedGas` like saving the bumped attempt. Maybe we don't even have to add this message to the error though. Before we would just return the error that came up during `tryAgainBumpingGas`. Maybe here we can just return `newErr`. Also we can maybe name this something more relevant like `replaceErr`?
```suggestion
		bumpedAttempt, retryable, replaceErr := eb.replaceAttemptWithBumpedGas(ctx, lgr, err, etx, attempt)
		if replaceErr != nil {
			return replaceErr, retryable
		}
```